src/assets/translations/en/main.json

```diff
@@ -985,6 +985,7 @@
     "foxPage": {
       "totalFoxValue": "Total Fox Value",
       "foxToken": "ShapeShift %{assetSymbol} Token",
+      "header": "FOX is an ERC-20 token created by ShapeShift which serves as the governance token for the ShapeShift DAO, token holders can vote on proposals relating to the operation...",
```

Comment:
What about getting the description from a selector?

Reply:
Isn't is possible to trim the one from selector with and ellipsis at the end?

Reply:
done !